setup.py

```diff
@@ -149,6 +149,7 @@ def run(self):
         'knack>=0.7.2',
         'markdown>=3.3.0',
         'matplotlib>=3.0.0',
+        'mpi4py>=3.1.3',
```

Comment:
I think you will also need to update cpu pipeline to install mpi before installing setup.py https://github.com/microsoft/superbenchmark/blob/b5c7c85d178cf4914bec4526bebd55a962f74de4/.azure-pipelines/cpu-unit-test.yml#L19
```sh
apt update
apt install -y libopenmpi-dev
```